pegasus/sites.v3/hourofcode.com/public/index.haml

```diff
@@ -224,52 +201,7 @@ social:
     // hoc2018: temporarily disable the cycling.
     //timerId = setInterval(cycleImage, 6000);
   });
-
-  function startVideo()
```

Comment:
It's nice to get rid of so much code, but do we think we'll be restoring it sooner or later?

Reply:
Stakeholders want this design as the evergreen design, even after Hour of Code. They've agreed that we have so many CTA at the top and they're okay with having the video permanently "downgraded" to next to the sign up form.